### PR TITLE
[Workers]: Update Form Submission Tutorial

### PR DESCRIPTION
**Pull request by lauragift21**
This PR updates the tutorial to use latest version of wrangler and module syntax format. 

Part of Quick-wins and #7610

### CODE REVIEW

**Review by kristianfreeman — APPROVED** [review]

File: content/workers/tutorials/handle-form-submissions-with-airtable/index.md [hunk 1]
```diff
@@ -25,7 +23,7 @@ In this tutorial, you will use Workers and [Airtable](https://airtable.com) to p
 
 [The front-end portion of this site](https://airtable-form-example.pages.dev) does not require any specific React experience. It uses a basic HTML5 form, showing that you can use Workers to handle any kind of form, whether entirely in HTML, or client-side, JavaScript-heavy forms, such as with React or other front-end frameworks.
 
-## Create a form
+## Step 1: Create a form
```
Comment (deadlypants1973) [inline content/workers/tutorials/handle-form-submissions-with-airtable/index.md]:
```suggestion
## 1. Create a form
```
Let's mirror the Get started 1., 2., 3.... format.

File: content/workers/tutorials/handle-form-submissions-with-airtable/index.md [hunk 2]
```diff
@@ -115,7 +113,7 @@ Code is provided as an example below, including the first `<input>`, to show tha
 
 If you would like to follow along with this example, you can directly copy the `<form>` code from the [`Form` component](https://github.com/cloudflare/workers-airtable-form/blob/main/frontend/src/Form.js) into your own project, or use the codebase and plug in your own serverless function, following the next section in the tutorial.
 
-## Create a serverless function
+## Step 2: Create a serverless function
```
Comment (deadlypants1973) [inline content/workers/tutorials/handle-form-submissions-with-airtable/index.md]:
```suggestion
## 2. Create a Worker
```

File: content/workers/tutorials/handle-form-submissions-with-airtable/index.md [hunk 3]
```diff
@@ -115,7 +113,7 @@ Code is provided as an example below, including the first `<input>`, to show tha
...
 
 To handle the form submission, you will create and deploy a Workers serverless function that parses the incoming form data and prepares it for submission to Airtable.
```
Comment (deadlypants1973) [inline content/workers/tutorials/handle-form-submissions-with-airtable/index.md]:
```suggestion
To handle the form submission, you will create and deploy a Worker that parses the incoming form data and prepares it for submission to Airtable.
```

File: content/workers/tutorials/handle-form-submissions-with-airtable/index.md [hunk 4]
```diff
@@ -115,7 +113,7 @@ Code is provided as an example below, including the first `<input>`, to show tha
...
```
Comment (deadlypants1973) [inline content/workers/tutorials/handle-form-submissions-with-airtable/index.md]: `wrangler init` is deprecated and these instructions should be replaced with the C3 instructions in the Get started: developers.cloudflare.com/workers/](https://developers.cloudflare.com/workers/get-started/guide/

File: content/workers/tutorials/handle-form-submissions-with-airtable/index.md [hunk 5]
```diff
@@ -135,20 +133,7 @@ If you have chosen to work with the [sample codebase on GitHub](https://github.c
...
-highlight: [4]
----
-name = "airtable-form-handler"
-type = "javascript"
-
-account_id = "yourAccountId"
-```
-
-## Configure an Airtable base
+## Step 3: Configure an Airtable base
```
Comment (deadlypants1973) [inline content/workers/tutorials/handle-form-submissions-with-airtable/index.md]:
```suggestion
## 3. Configure an Airtable base
```

File: content/workers/tutorials/handle-form-submissions-with-airtable/index.md [hunk 6]
```diff
@@ -206,30 +191,24 @@ Before you continue, review the keys that you should have from Airtable:
 2.  **Airtable Base ID:** the alphanumeric base ID found at the top of your base's API page.
 3.  **Airtable API Key:** the private API key found in example API requests on the Airtable API documentation page.
 
-## Submit data to Airtable
+## Step 4: Submit data to Airtable
```
Comment (deadlypants1973) [inline content/workers/tutorials/handle-form-submissions-with-airtable/index.md]:
```suggestion
## 4. Submit data to Airtable
```

File: content/workers/tutorials/handle-form-submissions-with-airtable/index.md [hunk 7]
```diff
@@ -206,30 +191,24 @@ Before you continue, review the keys that you should have from Airtable:
...
 
 With your Airtable base set up, and the keys and IDs you need to communicate with the API ready, you will now set up your Workers function and persist data from your form into Airtable.
 
-In `index.js`, begin by setting up a simple Workers handler that can respond to requests. When the URL requested has a pathname of `/submit`, you will handle a new form submission, otherwise, you will redirect to `FORM_URL`, a constant representing your front-end form URL (for example, [airtable-form-example.pages.dev](https://airtable-form-example.pages.dev)):
+In `index.js`, begin by setting up a Workers handler that can respond to requests. When the URL requested has a pathname of `/submit`, you will handle a new form submission, otherwise, you will redirect to `FORM_URL`, a constant representing your front-end form URL (for example, [airtable-form-example.pages.dev](https://airtable-form-example.pages.dev)):
```
Comment (deadlypants1973) [inline content/workers/tutorials/handle-form-submissions-with-airtable/index.md]: Per C3, 'index.js' would've been replaced by 'worker.js'.

**Review by deadlypants1973 — CHANGES_REQUESTED** [review]

**Review by deadlypants1973 — APPROVED** [review]
PCX

**Pull request merged**